### PR TITLE
attribute maxlength

### PR DESCRIPTION
**Pull request by darlanmendonca**
implementation and some specs to attribute maxlength, following specs from https://github.com/minimalist-components/webcomponents/issues/19

### CODE REVIEW

File: webcomponents/input/input.webcomponent.spec.js [hunk 1]
```diff
@@ -176,11 +176,30 @@ describe('mn-input (webcomponent)', () => {
     })
 
     it('should set empty text to label when attribute is removed', () => {
-      element.setAttribute('placeholder','test')
+      element.setAttribute('placeholder', 'test')
       element.removeAttribute('placeholder')
```
Comment (alencar13) [inline webcomponents/input/input.webcomponent.spec.js]: after to run this line, the attribute **is not being excluded**, as show below:

max_length

And even existing, the test is passing.

File: webcomponents/input/input.webcomponent.spec.js [hunk 2]
```diff
@@ -176,11 +176,30 @@ describe('mn-input (webcomponent)', () => {
...
+
+    it('should update attribute in child input', () => {
+      element.setAttribute('maxlength', '2')
+      element.setAttribute('maxlength', '3')
+      expect(element).to.contain('input').with.attribute('maxlength', '3')
+    })
+
+    it('should remove attribute from child input', () => {
+      element.setAttribute('maxlength', '2')
+      element.removeAttribute('maxlength')
```
Comment (alencar13) [inline webcomponents/input/input.webcomponent.spec.js:199]: after to run this line, the attribute is not being excluded, as show below:

max_length

And even existing, the test is passing.

File: webcomponents/input/input.webcomponent.spec.js [hunk 3]
```diff
@@ -176,11 +176,30 @@ describe('mn-input (webcomponent)', () => {
...
+      expect(element).to.contain('input').to.not.have.attribute('maxlength')
```
Comment (alencar13) [inline webcomponents/input/input.webcomponent.spec.js:200]: This verification is wrong, because maxlength attribute exists and the test is passing.

Comment (darlanmendonca) [reply webcomponents/input/input.webcomponent.spec.js:200]: you're right, I do a fix, please, review again the specs for attribute maxlength

**Review by alencar13 — APPROVED** [review]

**Comment by alencar13** [comment]
I think it's ok all the specs for this attribute.

**Pull request merged**